roxie/udplib/udptrr.cpp

```diff
@@ -127,7 +127,10 @@ class CReceiveManager : implements IReceiveManager, public CInterface
         UdpSenderEntry(const IpAddress &_dest, unsigned port) : dest(_dest)
         {
             SocketEndpoint ep(port, dest);
-            flowSocket = ISocket::udp_connect(ep);
+            flowSocket = ISocket::udp_connect(ep, udpFlowSocketsSize);
+            size32_t actualSize = flowSocket->get_send_buffer_size();
```

Comment:
trivial: This could be inside the if().  Same for other calls below.  (I have no idea how long the call takes)

Reply:
yes, of course, changed.